constants/users.js

```diff
@@ -1,7 +1,13 @@
-const userStatusEnum = ["ooo", "idle", "active"];
 const profileStatus = {
   PENDING: "PENDING",
   APPROVED: "APPROVED",
   NOT_APPROVED: "NOT APPROVED",
 };
-module.exports = { userStatusEnum, profileStatus };
+
+const userStatus = {
```

Comment:
Constants are usually uppercase
```suggestion
const USER_STATUS = {
```